tests/conftest.py

```diff
@@ -188,6 +188,10 @@ def _create_role(do_not_mark=False, **args):
                 "manage_template_metric_sets": False,
                 "manage_renaming_rules": False,
                 "manage_template_checklists": False,
+                "manage_audit_logs": False,
+                "manage_template_structure_sets": False,
+                "manage_workspace_algorithms": False,
+                "organization_manage_access_patients": False,
```

Comment:
I'd just like to point out that the tests should be passing without having to add this. If they are not, this still needs to be addressed in ProKnow, not the SDK.